README.md

```diff
@@ -1,155 +1,189 @@
-<!-- [![Build and Test](https://github.com/dhiway/cord/workflows/Build%20and%20Test/badge.svg)](https://github.com/dhiway/cord/actions) -->
+This repository contains the Rust implementation of a [Cord Network][cord-homepage] node based on the [Substrate][substrate-homepage] framework.
 
 # CORD
 
-The CORD node implementation uses Parity Substrate as the underlying
-technology stack with custome pallets and hierarchical trust Modules.
+Cord is designed to be a global public utility and trust framework that creates new possibilities in how we address trust gaps, manage transactions and exchanges of value at scale.
 
-- [CORD](#cord)
-  - [Accounts](#accounts)
-  - [Build & Run](#build--run)
-    - [Building in dev mode](#building-in-dev-mode)
-    - [Building in performant release mode](#building-in-performant-release-mode)
-    - [Start the Node, by running:](#start-the-node-by-running)
-    - [Debug Mode](#debug-mode)
-    - [Release Mode](#release-mode)
-    - [Setup the local node with session keys, by running](#setup-the-local-node-with-session-keys-by-running)
-  - [CORD User Accounts](#cord-user-accounts)
-    - [Polkadot{.js} Browser Plugin](#polkadotjs-browser-plugin)
-    - [Polkadot Web UI](#polkadot-web-ui)
-  - [Building docker image](#building-docker-image)
-  - [Node Modules functionalities](#node-modules-functionalities)
-    - [Substrate Documentation](#substrate-documentation)
-    - [Substrate Tutorials](#substrate-tutorials)
-    - [Substrate JSON-RPC API](#substrate-json-rpc-api)
-    - [Substrate Reference Rust Docs](#substrate-reference-rust-docs)
+It is designed to simplify the management of information, making it easier for owners to control; agencies and businesses to discover, access and use data to deliver networked public services. It provides a transparent history of information and protects it from unauthorised tampering from within or without the system.
 
-### Accounts
+Cord leverages the modular approach of Substrate framework and defines a rich set of primitives to catalyse exceptional levels of innovation in both existing and emerging industries-from conducting transactions and maintaining records, across many sectors of the economy such as finance, trade, health, energy, water, resources, agriculture and credentials.
```

Comment:
CORD builds on the modular approach of Substrate framework. It defines a rich set of primitives to help design exceptional levels of innovations for existing and emerging industries. Such innovative approaches cover conducting transactions and maintaining records across a number of sectors such as finance, trade, health, energy, water resources, agriculture and related credentials.